run.py

```diff
@@ -416,7 +416,8 @@ def fit(self):
 				if kill_cnt % 10 == 0 and self.p.gamma > 5:
 					self.p.gamma -= 5 
 					self.logger.info('Gamma decay on saturation, updated value of gamma: {}'.format(self.p.gamma))
-				if kill_cnt > 25:
+
+				if self.p.enable_early_stopping & kill_cnt > self.p.early_stopping_limit:
```

Comment:
Guess you meant "and" (the boolean operator) and not the bit-wise and (&) ?
```suggestion
				if self.p.enable_early_stopping and kill_cnt > self.p.early_stopping_limit:
```

Reply:
Yep thanks